src/style.css

```
@@ -0,0 +1,35 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+p,
+h1,
+h2 {
+  margin: 0;
+}
+
+a:link ,
+a:visited,
+a:active,
+a:hover {
+  color: green;
+}
+
+.mainarticle {
```

Якщо клас складається з двух слів, то розділяємо їх дефісом
```suggestion
.main-article {
```